finp2p-contracts/contracts/token/ERC20/FINP2POperatorERC20.sol

```diff
@@ -189,6 +205,7 @@ contract FINP2POperatorERC20 is IFinP2PAsset, IFinP2PEscrow, AccessControl {
         uint256 quantity,
         uint256 expiry,
         bytes32 assetHash,
+        string memory assetType,
```

Comment:
This was planned to fix on another PR, but if you already did, please create an issue to describe it

Reply:
Consider using enum of asset types instead

Reply:
ok